workshops/geometric_pattern/README.md

```diff
@@ -1,15 +1,22 @@
 # Geometric Pattern
 
-![](img/sample1.png)
+| What you'll build    |
+| -------------------- |
+| ![](img/sample1.png) |
 
-Short link to this workshop: https://workshops.hackclub.com/geometric_pattern
+Here's a link to the [live demo][live_demo] and the [final code][final_code]. This workshop should take around 1 hour.
 
-[Demo](http://prophetorpheus.github.io/geometric_pattern/)
+_**Warning: this demo involves flashing colors and is not recommended for those with a history of epilepsy.**_ There's also an [animated demo][animated_live_demo].
+
+_**We recommend going through this workshop in Google Chrome.**_
+
+[live_demo]: https://prophetorpheus.github.io/geometric_pattern
+[animated_live_demo]: https://prophetorpheus.github.io/geometric_pattern/version2.html
+[final_code]: https://github.com/prophetorpheus/prophetorpheus.github.io/tree/master/geometric_pattern
 
 ---
 
-We'll be creating a cool graphic like the one above, using JavaScript and p5.js.
-[p5.js](http://p5js.org/) is a library for making stuff in conjunction with the [HTML canvas element](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/canvas). p5.js is nothing more than JavaScript code written to make commonly-desired functionality much more accessible. It is primarily used to make visuals and graphics. A great introduction can be found [here](http://taeyoonchoi.com/signing-coders-1/).
+We'll be creating a cool graphic like the one above, using JavaScript and p5.js. [p5.js](http://p5js.org/) is a library for making stuff in conjunction with the [HTML canvas element](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/canvas). p5.js is nothing more than JavaScript code written to make commonly-desired functionality much more accessible. It is primarily used to make visuals and graphics. A great introduction can be found [here](http://taeyoonchoi.com/signing-coders-1/).
```

Comment:
I think this link regarding the introduction to p5 should have more information associated with it, for example: Who is it intended to be used by, how long will it take?